src/modules/app/route-configs/TunewsRouteConfig.js

```diff
@@ -28,8 +25,8 @@ const tunewsRoute: Route = {
     await Promise.all([
       fetchData(createCitiesEndpoint(cmsApiBaseUrl), dispatch, state.cities),
       fetchData(createEventsEndpoint(cmsApiBaseUrl), dispatch, state.events, { city, language }),
-      fetchData(createTunewsLanguagesEndpoint(tunewsApiBaseUrl, true), dispatch, state.languages, { city, language }),
-      fetchData(createTunewsEndpoint(tunewsApiBaseUrl), dispatch, state.tunews, { page: 1, language: language, count: 20 })
+      fetchData(createTunewsLanguagesEndpoint(tunewsApiBaseUrl), dispatch, state.languages, { city, language }),
```

Comment:
Are these even used?

Reply:
Wow... It seems we "just" override `state.languages` (the paylaod for the languages of the current city)... Not sure what impact this has and if that's what we want...

Reply:
Oh yeah nice. Hmm I guess we don't want this, even though I think right now this is no problem. I have no overview over the code though. Could you create an issue if you think this should be changed please?

Reply:
@Schedulaar what do you mean?

Reply:
I think this is fine :+1: